### PR TITLE
Add user agent to requests in python scripts

### PR DESCRIPTION
**Pull request by DaoWen**
Clarifies the origin of some requests in the cook scheduler access logs.

### CODE REVIEW

File: cli/cook/http.py [hunk 1]
```diff
@@ -29,6 +31,7 @@ def configure(config):
     http_adapter = adapters_module.HTTPAdapter(max_retries=retries)
     session = session_module.Session()
     session.mount('http://', http_adapter)
+    session.headers['User-Agent'] = f'CookCLI/{cook.version.VERSION}'
```
Comment (dposada) [inline cli/cook/http.py]: I would favor `cs/...`

Comment (dposada) [inline cli/cook/http.py]: Can we bump the version of the CLI for this change?

File: cli/cook/version.py [hunk 2]
```diff
@@ -1 +1 @@
-VERSION = '0.7.0'
+VERSION = '0.7.1'
```
Comment (DaoWen) [inline cli/cook/version.py]: Bumped Cook CLI version number.

Comment (dposada) [reply cli/cook/version.py]: We're already at 0.7.1 on master:

https://github.com/twosigma/Cook/blob/master/cli/cook/version.py

File: cli/cook/http.py [hunk 3]
```diff
@@ -29,6 +31,7 @@ def configure(config):
     http_adapter = adapters_module.HTTPAdapter(max_retries=retries)
     session = session_module.Session()
     session.mount('http://', http_adapter)
+    session.headers['User-Agent'] += f' (cs/{cook.version.VERSION})'
```
Comment (DaoWen) [inline cli/cook/http.py]: Name changed: `CookCLI` → `cs`

I also decided that it made more sense to append the extra user agent info to the default user-agent header of the response library. Now it looks like this in the logs: `"python-requests/2.18.4 (cs/0.7.1)"`

Comment (shamsimam) [reply cli/cook/http.py]: Can we change the order to `cs/0.7.1 (python-requests/2.18.4)`? This makes more sense to me as `cs` is what is more important, that it uses `python-request` is an implementation detail.

Comment (shamsimam) [reply cli/cook/http.py]: Based on https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent , `cs` is our product, `0.7.1` is the version and `(python-requests/2.18.4)` is system information.

Comment (DaoWen) [reply cli/cook/http.py]: OK, you convinced me. I'll reverse it.

File: cli/cook/version.py (hunk 2 above)
Comment (DaoWen) [reply cli/cook/version.py]: Rebased (and bumped to 0.7.2).

**Pull request merged**